int32   xMask
int32   yMask
float64 arena_rvec_0
float64 arena_rvec_1
float64 arena_rvec_2
float64 arena_tvec_0
float64 arena_tvec_1
float64 arena_tvec_2
